particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,161 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'GameSchemas.arcs'
+import 'Stores.arcs'
+
+particle Game in './source/Game.js'
```

Comment:
The names of *.arcs files, Particles, and Recipes tend to go into giant lists. Names like "Demo", "Game", and so on are possibly too generic, and make it hard to differentiate the artifacts.

It's a bit of a hassle, but in, e.g., TVMaze/*, notice all the artifacts are prefixed with TVMaze to address this problem.